holmes/core/tool_calling_llm.py

```
@@ -165,16 +167,68 @@ class ToolCallingLLM:
+        model_name = (self.llm.model or "").lower()
+
+        try:
+            provider_info = litellm.get_llm_provider(model_name)
+            if (
+                provider_info is not None
+                and len(provider_info) > 1
+                and provider_info[1] in {"anthropic", "bedrock"}
+            ):
+                return True
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)